src/get-started/editor.md

```diff
@@ -41,92 +50,128 @@ that's OK, skip ahead to the [next step: Test drive][].
 
 ## Install VS Code
 
-VS Code is a lightweight editor with complete Flutter
-app execution and debug support.
+[VS Code][] is a code editor to build and debug apps.
+With the Flutter extension installed, you can compile, deploy, and debug
+Flutter apps.
+
+To install the latest version of VS Code,
+follow Microsoft's instructions for the relevant platform:
+
+- [Install on macOS][]
+- [Install on Windows][]
+- [Install on Linux][]
+
+[VS Code]: https://code.visualstudio.com/
+[Install on macOS]: https://code.visualstudio.com/docs/setup/mac
+[Install on Windows]: https://code.visualstudio.com/docs/setup/windows
+[Install on Linux]: https://code.visualstudio.com/docs/setup/linux
+
+## Install the VS Code Flutter extension
+
+1. Start **VS Code**.
+
+1. Open a browser and go to the [Flutter extension][] page
+   on the Visual Studio Marketplace.
+
+1. Click **Install**.
+   Installing the Flutter extension also installs the Dart extension.
+
+[Flutter extension]: https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
+
+## Validate your VS Code setup
+
+1. Go to **View** <span aria-label="and then">></span> **Output**.
+
+   You can also press <kbd>Control</kbd> / <kbd>Command</kbd> +
+   <kbd>Shift</kbd> + <kbd>U</kbd>.
 
-* [VS Code][], latest stable version
+1. In the dropdown on the upper right of the **Output** panel, 
+   select **flutter (flutter)**.
 
-## Install the Flutter and Dart plugins
+1. Go to **View** <span aria-label="and then">></span>
+   **Command Palette...**.
 
- 1. Start VS Code.
- 1. Invoke **View > Command Palette...**.
- 1. Type "install", and select
-    **Extensions: Install Extensions**.
- 1. Type "flutter" in the extensions search field,
-    select **Flutter** in the list, and click **Install**.
-    This also installs the required Dart plugin.
+   You can also press <kbd>Control</kbd> / <kbd>Command</kbd> +
+   <kbd>Shift</kbd> + <kbd>P</kbd>.
 
-## Validate your setup with the Flutter Doctor
+1. Type `doctor`.
 
- 1. Invoke **View > Command Palette...**.
- 1. Type "doctor", and select the
-    **Flutter: Run Flutter Doctor**.
- 1. Review the output in the **OUTPUT** pane for any issues.
-    Make sure to select Flutter from the dropdown
-    in the different Output Options.
+1. Select the **Flutter: Run Flutter Doctor**.
+   Flutter Doctor runs and its response displays in the **Output** panel.
 
 </div>
 <div class="tab-pane" id="androidstudio" role="tabpanel" aria-labelledby="androidstudio-tab" markdown="1">
 
-## Install Android Studio
+## Install Android Studio or IntelliJ IDEA
 
-Android Studio offers a complete,
-integrated IDE experience for Flutter.
+Android Studio and IntelliJ IDEA offer a complete,
+IDE experience once you install the Flutter plugin.
 
-* [Android Studio][], version 2020.3.1 (Arctic Fox) or later
+To install one of the following IDEs, follow their instructions:
 
-Alternatively, you can also use IntelliJ:
+- [Android Studio][], version 2022.2 or later
```

Comment:
Since Android Studio releases are often known by their codename, consider adding that back in:

```suggestion
- [Android Studio][], version 2022.2 (Flamingo) or later
```

Reply:
Thanks for this! I didn't know the codename.